installer/conf/syslog-ng-lad.conf

```diff
@@ -0,0 +1,31 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%) (LAD_oms marker for unconfig)
+# To be added for each facility/min-level: log { source(src); filter(f_LAD_oms_f_<facility>); filter(f_LAD_oms_ml_<min_level>); destination(d_LAD_oms); };
+destination d_LAD_oms { udp("127.0.0.1" port(%SYSLOG_PORT%)); };
```

Comment:
No problem. Moved.

I don't have a log line here, because I don't want rsyslog/syslog-ng to start sending logs to the configured ports when there's no omsagent started (remember I don't start it when onboarding). Only after LAD starts omsagent using service_control start LAD, LAD will configure rsyslog/syslog-ng (with log lines) and restart syslog.